examples/odop_solar.py

```diff
@@ -1,4 +1,5 @@
 import taichi as ti
+import math
```

Comment:
Please file a separate PR, I don't think these changes are trivially off-the-topic.